### ISSUE TITLE
Doesn't work with Youtube Music

### ISSUE THREAD

**Issue by jorgemcs21** [Cilenco/skipTrackLongPressVolume#2]
Phone: Google Pixel 3
OS: Android Pie 9 (april update)

Issue: It works with Play Music, but doesn't with YT Music. I saw a comment displaying the same issue on the XDA developers article regarding this app, but the user had Android Q 10.0.

**Comment by JaredTheWolf** [comment]
Can confirm issue -- logcat attached. The app does seem to be doing its job...
[logcat.log](https://github.com/Cilenco/skipTrackLongPressVolume/files/3073628/logcat.log)

**Comment by LucianDevran** [comment]
Yeah same here, the app works with my standard music player but not with the YouTube Music app. Android 8.1.0 on an LG
